modules/file-extension-common/src/main/java/org/mule/runtime/module/extension/file/api/FileContentWrapper.java

```diff
@@ -130,7 +130,7 @@ private Object tryTransform(DataType sourceDataType, DataType targetDataType)
         Transformer transformer;
         try
         {
-            transformer = event.getMuleContext().getRegistry().lookupTransformer(sourceDataType, targetDataType);
+            transformer = castEvent().getMuleContext().getRegistry().lookupTransformer(sourceDataType, targetDataType);
```

Comment:
Can you add the MuleContext as a constructor parameter of this class instead?. We will likely remove the MuleContext from the MuleEvent.